app-frontend/src/assets/styles/sass/_shame.scss

```diff
@@ -3636,11 +3655,90 @@ rf-aoi-draw-toolbar {
     display: flex;
 }
 
-.analysis-viz-container {
-    padding: 1rem 3rem 1rem 0rem !important;
-}
-
 .panel-analysis-viz-container {
     background-color: $white;
     box-shadow: 0px 3px 8px rgba($shade-normal, 0.2);
 }
+
+rf-data-viz-histogram {
+    position: relative;
+    display: block;
+    width: 100%;
+    .data-fill {
+        fill: #465076 !important;
+        fill-opacity: 1 !important;
+    }
+    ng-transclude {
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        position: absolute;
+    }
+}
+
+rf-histogram-bar {
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    position: absolute;
+
+    div {
+        height: 50px;
+        width: 2px;
+        position: absolute;
+    }
+}
+
+.histogram-bar-gray {
```

Comment:
can we just do these as generic color/background colors in _helpers.scss? Could ask @designmatty what the best way to handle this is.